tests/unit/testdata/invalid_key-os-dependent_packages.json

```diff
@@ -0,0 +1,46 @@
+{
+  "invalid_key": [
+    {
+      "name": "cryptography",
+      "version": "41.0.5",
+      "platform": "win_amd64",
+      "python-version": "37",
```

Comment:
How will look pulling py37 and py39 dependencies? For example we want support upgrade to latest TA running older versions of Splunk until we introduce py39 breaking changes.

Reply:
that's actually a great point @dkvashninsplunk 

I'll ask around and let you know

Reply:
I'm not sure if this is what you're asking, but this is where the developer decides what gets downloaded and where. This version here allows pip to identify the wheel that we want to use. If we want to download the library for both p37 and p39, we simply add another element to the config with python-version:39 and target where we want to install it. This way we can have two directories e.g. windows/python37 and windows/python39 with the appropriate version of given library.